eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -1,3 +1,6 @@
+parameters:
+  DiffScriptPath: ''
```

Comment:
This appears unused

Reply:
Sorry. Am still testing. https://github.com/Azure/azure-sdk-for-net/pull/27338